src/litserve/specs/openai.py

```diff
@@ -77,6 +77,19 @@ class ImageContent(BaseModel):
     image_url: Union[str, ImageContentURL]
 
 
+class InputAudio(BaseModel):
+    data: str
+    """Base64 encoded audio data."""
```

Comment:
this is the way to document the entire class (in which case it'd have to be above data). If you want to document the data attribute only, the docstring should be a inline comment

Reply:
Sure @justusschock, thanks so much for the clarification! 
I’ve updated it accordingly.